enterprise/models.py

```diff
@@ -403,6 +403,13 @@ class Meta:
             "If checked, the learners will be able to see the academies on the learner portal dashboard."
         )
     )
+    enable_one_academy = models.BooleanField(
+        verbose_name="Enable One Academy feature",
+        default=False,
+        help_text=_(
+            "If checked, search will be replaced with One Academy feature."
```

Comment:
Can we more details? Currently the context is unclear. Something like

`If checked, then learners will only see Academy detail page on enterprise learner portal`

Reply:
@muhammad-ammar I initially had this same message, but One Academy only replaces search page, user will still be able to view all other pages like, Dashboard, course, program details pages etc.